packages/language-server/src/lintWorker.ts

```diff
@@ -1,6 +1,22 @@
-import { lintCypherQuery } from '@neo4j-cypher/language-support';
+import {
+  DbSchema,
+  lintCypherQuery as _lintCypherQuery,
+  _internalFeatureFlags,
+} from '@neo4j-cypher/language-support';
 import workerpool from 'workerpool';
 
+function lintCypherQuery(
+  query: string,
+  dbSchema: DbSchema,
+  featureFlags: { cypher25?: boolean } = {},
+) {
+  // We allow to override the consoleCommands feature flag
+  if (featureFlags.cypher25 !== undefined) {
+    _internalFeatureFlags.cypher25 = featureFlags.cypher25;
+  }
```

Comment:
We need to pass the feature flags to the lint worker because the `_internalFeatureFlags` variable from the outside lives in another thread